src/instructlab/model/train.py

```diff
@@ -304,6 +305,28 @@ def train(
     Takes synthetic data generated locally with `ilab data generate` and the previous model and learns a new model using the MLX API.
     On success, writes newly learned model to {model_dir}/mlx_model, which is where `chatmlx` will look for a model.
     """
+
+    if not legacy and os.path.isdir(data_path):
+        # need to get msgs jsonl, this is a hack but will work for now
+        pattern = os.path.join(data_path, f'*{"messages"}*')
```

Comment:
It seems like this isn't searching for `.jsonl` files but rather anything matching `*messages*`. So `messages.jpg` would match but `dataset.jsonl` wouldn't. We should fix the search pattern here to look for `*.jsonl` files if that's the intention.

Reply:
good point, thanks

Reply:
Should this be looking instead for `*messages*.jsonl` if that's what is wanted?

Reply:
Agreed the filter should be more explicit.